client/league.py

```diff
@@ -34,4 +38,33 @@ async def display_best_champs(message : Message) -> None:
         bestchamps_message += ", ".join([cm.champion.name for cm in good_with])
         await message.channel.send(bestchamps_message)
         Logger.log("Outputting Best Champ")
-    
\ No newline at end of file
+    
+    @staticmethod
+    async def display_match_history(message : Message) -> None:
+        summoner_name = message.content[14:]
+        summoner = Summoner(name=summoner_name, region="NA")
+        history = summoner.match_history[:5]
+        output = "{:>8} {:>14} {:>14} {:>14} {:>15} {:>16}".format(
+                                                    "MODE",
+                                                    "CHAMPION",
+                                                    "KILLS",
+                                                    "DEATHS",
+                                                    "ASSISTS",
+                                                    "KDA") + "\n" 
+        for match in history:
```

Comment:
this loop should be in league helper file.

Reply:
Thanks for the catch!